server/api/screenshot/screenshotController.js

```diff
@@ -15,33 +15,35 @@ exports.list = function(req, res, next){
   });
 };
 
+/**
+ * create
+ * ======
+ * Takes screenshot of specified URL, creates new Screenshot in DB, and
+ * adds screenshot to the user images.
+ */
 exports.create = function(req, res, next) {
-  var url = req.body.url;
   var username = req.params.username;
-  var annotatedImage = req.body.annotatedImage;
```

Comment:
if we remove annotatedImage variable, we should remove it also from the Screenshot creation on line 27 and also remove it from being required on the db